src/PVRFilmonData.cpp

```diff
@@ -333,6 +359,27 @@ PVR_ERROR PVRFilmonData::GetRecordings(ADDON_HANDLE handle) {
 	return PVR_ERROR_NO_ERROR;
 }
 
+PVR_ERROR PVRFilmonData::GetRecordingStreamProperties(const PVR_RECORDING* recording, PVR_NAMED_VALUE* properties, unsigned int* iPropertiesCount) {
+  P8PLATFORM::CLockObject lock(m_mutex);
```

Comment:
You are using a lock here, but not in  `PVRFilmonData::GetChannelStreamProperties`. Is that by intention?